cmd/loop/liquidity.go

```diff
@@ -338,18 +338,18 @@ func setParams(ctx *cli.Context) error {
 		flagSet = true
 	}
 
-	if ctx.IsSet("autoout") {
-		params.AutoLoopOut = ctx.Bool("autoout")
+	if ctx.IsSet("autoloop") {
```

Comment:
nit: This parameter is referenced in `autoloop.md`, line 9.

Reply:
fixed!